run/CCCX.X-M-V1/RAMSIN.INCUS.V1

```diff
@@ -0,0 +1,855 @@
+   ISSTFN   = '/nobackup/pmarines/rams_topo/sst-h5/S',         !CHANGEME
+              '/nobackup/pmarines/rams_topo/sst-h5/S',         !CHANGEME
+              '/nobackup/pmarines/rams_topo/sst-h5/S',         !CHANGEME
+   IVEGTFN  = '/nobackup/pmarines/rams_topo/ogedata-h5/GE',    !CHANGEME
+              '/nobackup/pmarines/rams_topo/ogedata-h5/GE',    !CHANGEME
+              '/nobackup/pmarines/rams_topo/ogedata-h5/GE',    !CHANGEME
+   ISOILFN  = '/nobackup/pmarines/rams_topo/FAOdata-h5/FAO',   !CHANGEME
+              '/nobackup/pmarines/rams_topo/FAOdata-h5/FAO',   !CHANGEME
+              '/nobackup/pmarines/rams_topo/FAOdata-h5/FAO',   !CHANGEME
+   NDVIFN   = '/nobackup/pmarines/rams_topo/ndvi-h5/',         !CHANGEME
+              '/nobackup/pmarines/rams_topo/ndvi-h5/',         !CHANGEME
+              '/nobackup/pmarines/rams_topo/ndvi-h5/',         !CHANGEME
+ 
+! Topography scheme
+
+   ITOPSFLG = 3,3,3,           ! 0 = Average Orography
+                                 ! 1 = Silhouette Orography
+                                 ! 2 = Envelope Orography
+                                 ! 3 = Reflected Envelope Orography
+
+   TOPTENH  = 1.,1.,1.,1.,       ! For ITOPSFLG=1, Weighting of topo
+                                 !   silhouette averaging 
+                                 ! For ITOPSFLG=2 or 3, Reflected Envelope
+                                 !   and Envelope Orography enhancement factor
+                                 
+   TOPTWVL  = 4.,4.,4.,4.,       ! Topo wavelength cutoff in filter 
+                                
+! Surface Roughness scheme
+
+   IZ0FLG   = 0,0,0,0,           ! 0 = Use veg, bare soil and water surface
+                                 ! 1 = Also use subgrid scale topography
+   Z0MAX    = 5.,5.,5.,5.,       ! Max zo for IZ0FLG=1
+   Z0FACT   = 0.005,             ! Subgrid scale orograhic roughness factor
+   
+ $END
+ 
+ $MODEL_OPTIONS
+
+! Numerical schemes
+ 
+   ICORFLG  = 1,             ! Coriolis flag/2D v-component  -  0=off, 1=on
+
+   IBND     = 1,             ! Lateral boundary condition flags
+   JBND     = 1,             ! 1-Klemp/Wilhelmson, 2-cyclic
+
+   CPHAS    = 30.,           ! Phase speed if IBND or JBND = 1
+   LSFLG    = 2,             ! Large-scale gradient flag for variables other than
+                             !  normal velocity:
+                             !  0 = zero gradient inflow and outflow
+                             !  1 = zero gradient inflow, radiative b.c. outflow
+                             !  2 = constant inflow, radiative b.c. outflow
+                             !  3 = constant inflow and outflow
+   NFPT     = 0,             ! Rayleigh friction - number of points from the top
+   DISTIM   = 60.,           !                   - dissipation time scale
+
+   ISPONGE_PTS = 0,5,8,      !  Number of nested grid boundary sponge points !CHANGEME 
+   SPONGE_TAU  = 30.,        !  Nested grid sponge zone time scale (seconds)
```

Comment:
SPONGE_TAU needs to have 3 entries, needs to match the # of grids used
bin.rams/RAMSIN.testrunonly lists:
SPONGE_TAU = 0., 30., 30.,

Reply:
Done!